Source/Extensions/Blazorise.Video/Utilities/VideoParsers.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace Blazorise.Video.Utilities;
+
+internal static class VideoParsers
+{
+    public static double? ParseAspectRatio( string aspectRatio )
+    {
+        if ( string.IsNullOrEmpty( aspectRatio ) || !aspectRatio.Contains( ':' ) )
+        {
+            return null;
```

Comment:
Why does this one not warrant a special message? Specially the` !aspectRatio.Contains( ':' )`

Reply:
Because in Plyr.js, Ratio is defined as a string, e.g., `"16:9"`, while in the new library, it needs to be described as a number `16 / 9`. So to not add a breaking change I just parse and convert it to a number.

Reply:
Also it is of note that we usually don't typify/return exceptions and just silently return null. Not sure if we want to be consistent with that or not.

Reply:
I am more open to going the silent route than raising an exception.